scss/style.scss

```diff
@@ -66,6 +66,64 @@
         }
     }
 
+    .lp-json-pollock-layout-form {
+        .lp-json-pollock-element-text {
+            margin: 2.5%;
+        }
+
+        .lp-json-pollock-layout-checklist {
+            border: none;
+            margin: 5% 0%;
+
+            > .lp-json-pollock-element-checkbox {
+                padding: 2px 0px;
+
+                & > .lp-json-pollock-border-element {
+                    border-top: solid 0.5px #d4d4d5;
```

Comment:
use variables - use `$default-border` and override its width - do not add explicit colors and sizes